lib/constant.mli

```diff
@@ -152,6 +161,9 @@ val of_symbolic_data : symbolic_data -> ('scalar,'pte,'instr) t
 val as_pte : ('scalar,'pte,'instr) t -> ('scalar,'pte,'instr) t option
 val is_pt : ('scalar,'pte,'instr)  t -> bool
 
+(* Remove the Pac field of a virtual address *)
+val strip_pauth : ('scalar,'pte,'instr) t -> ('scalar,'pte,'instr) t
```

Comment:
Really optional, I just wondered if there is any opportunity to reuse code between this and the logic for "SetCanonical" in symbValue.ml. Also, maybe this could be named "strip_pac" (or something in sync with the setcanonical)? It's just that "pauth" sounds like the name of the feature.